dev-docs/tutorials/getting-started.md

```diff
@@ -0,0 +1,142 @@
+# Get started developing for Ubuntu Pro Client
+
+This tutorial will walk you through getting your development environment set up from scratch for contributing to Ubuntu Pro Client.
+
+We generally support the latest interim release and the latest LTS of Ubuntu as development hosts for Ubuntu Pro Client.
+
+> **Note**
+> This tutorial was last tested on Ubuntu 23.10 Mantic Minotaur (mantic).
+
+# From nothing to unit tests
+Open a terminal.
+
+Install necessary host tooling.
+```bash
+sudo apt install git tox pre-commit intltool libapt-pkg-dev sbuild-launchpad-chroot
+```
+> **Note**
+> Python comes installed by default on Ubuntu.
+> Do not install python via any non-Ubuntu means. That includes conda, pyenv, asdf, ppa:deadsnakes/ppa, download from python.org, etc.
+
+Get the code.
+```bash
+git clone https://github.com/canonical/ubuntu-pro-client.git
+cd ubuntu-pro-client
+```
+
+Initialize `pre-commit`.
+```bash
+pre-commit install
+```
+
+Run the unit tests and other checks.
+```bash
+tox
+```
+
+> **Note**
+> You can run individual jobs (or "environments" in tox terminology) with the `-e` flag.
+>
+> For example, `tox -e test` will only run the unit tests.
+>
+> Explore `tox.ini` to see what tox environments are available and how they're configured.
+
+# Getting to basic behave tests
+We use [`behave`](https://behave.readthedocs.io/en/stable/) as a framework for writing integration/acceptance/regression tests. behave tests are in the `features/` folder and generally follow the pattern of:
+1. Launch a VM or container on a platform
```

Comment:
I was going for a generic description of what the behave test does, so its kinda vague for a reason. And I don't want to list all of the supported platforms here because it will get out of date. I do like the idea of listing a couple examples in parentheses though to demonstrate what we mean by "platform", I'll add that for sure